drivers/docker/handle.go

```diff
@@ -308,6 +308,9 @@ func (h *taskHandle) run() {
 		h.logger.Error("failed to wait for container; already terminated")
 	}
 
+	ctx, inspectCancel := context.WithTimeout(context.Background(), 10*time.Second)
+	defer inspectCancel()
```

Comment:
The `dockerTimeout` of 5m seems ridiculous for these two API calls, which should return immediately as far as I can tell. Even 10s seems like a lot, but I don't really know what we can expect out of Docker's API here.